assignments/blog/server/database/db.sql

```diff
@@ -2,12 +2,48 @@ CREATE DATABASE blog;
 
```

Comment:
db.sql - add command to undo database creation to prevent duplicate data being inserted
README.md - add instructions for user to run db.sql to set up database